### ISSUE TITLE
Inaccessible host: `ce.ap-southeast-1.amazonaws.com` at port `undefined`

### ISSUE THREAD

**Issue by semsphy** [kamranahmedse/aws-cost-cli#1]
## Current Behavior

```bash
aws-cost --profile NAME
```

## Expected Behavior

Display the cost of the profile `NAME`

## Debug log

```
Getting pricing data/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/event_listeners.js:611
        this.response.error = AWS.util.error(new Error(message), {
                                             ^

UnknownEndpoint: Inaccessible host: `ce.ap-southeast-1.amazonaws.com' at port `undefined'. This service may not be available in the `ap-southeast-1' region.
    at Request.ENOTFOUND_ERROR (/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/event_listeners.js:611:46)
    at Request.callListeners (/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/request.js:686:14)
    at error (/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/event_listeners.js:443:22)
    at ClientRequest.<anonymous> (/opt/homebrew/lib/node_modules/aws-cost-cli/node_modules/aws-sdk/lib/http/node.js:99:9)
    at ClientRequest.emit (node:events:513:28)
    at ClientRequest.emit (node:domain:489:12)
    at TLSSocket.socketErrorListener (node:_http_client:494:9)
    at TLSSocket.emit (node:events:513:28) {
  code: 'UnknownEndpoint',
  region: 'ap-southeast-1',
  hostname: 'ce.ap-southeast-1.amazonaws.com',
  retryable: true,
  originalError: Error: getaddrinfo ENOTFOUND ce.ap-southeast-1.amazonaws.com
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26) {
    errno: -3008,
    code: 'NetworkingError',
    syscall: 'getaddrinfo',
    hostname: 'ce.ap-southeast-1.amazonaws.com',
    region: 'ap-southeast-1',
    retryable: true,
    time: 2023-01-23T02:18:55.422Z
  },
  time: 2023-01-23T02:18:55.422Z
}

Node.js v18.11.0
```

**Comment by andrevdm** [comment]
This workaround lets me generate the report: I changed my region in the aws config file (`region = us-east-1`) and then it works fine. AFAIK cost explorer must be run on the `us-east-1` endpoint. Its global so it will find your `ap-southeast-1` services.

**Comment by kamranahmedse** [comment]
@semsphy can you please confirm if [you have enabled Cost Explorer](https://docs.aws.amazon.com/cost-management/latest/userguide/ce-enable.html)?

**Comment by jawobar** [comment]
I am facing the same issue with Cost Explorer enabled and eu-central-1 region.

```
  hostname: 'ce.eu-central-1.amazonaws.com',
  retryable: true,
  originalError: Error: getaddrinfo ENOTFOUND ce.eu-central-1.amazonaws.com```

**Comment by kamranahmedse** [comment]
This has been fixed in `0.2.4`. Retry by updating the version:

```
npm install aws-cost-cli --global
```

We now default to `us-east-1` but you can forcefully change the region by using `-r` option.

**Comment by jawobar** [comment]
Works for me! Thanks for the quick fix.
